frontend/public/components/overview/pods-overview.tsx

```diff
@@ -70,14 +72,21 @@ const isPodWithoutImageId = (pod: PodKind) =>
   pod.status?.phase === 'Pending' &&
   pod.status?.containerStatuses?.some((containerStatus) => !containerStatus.imageID);
 
-const podCompare = (pod1: PodKind, pod2: PodKind): number => {
+export const podCompare = (pod1: PodKind, pod2: PodKind): number => {
   const error1 = isPodError(pod1);
   const error2 = isPodError(pod2);
 
   if (error1 !== error2) {
     return error1 ? 1 : 0;
   }
```

Comment:
Please change this similar to your order to:

```suggestion
  if (error1 !== error2) {
    return error1 ? -1 : 1;
  }
```

Then the error cases are ordered correctly as well!

Your new ticket is still good to re-validate this and check why terminated pods are not shown at the top (I now already that there is no status update when a pod is marked as deleted. We should respect the `deletionTimestamp` there as well. But we can check the details in the other ticket. With or withour PM/UX.)

Reply:
Updated.